docs/troubleshooting.md

```diff
@@ -0,0 +1,66 @@
+# Troubleshooting
+
+## Enable Debug Logging
+
+When things are not working, a good first step is to restart the program with
+debug logging enabled. Do this by setting the `OTEL_LOG_LEVEL` environment
+variable to `debug`.
+
+```sh
+export OTEL_LOG_LEVEL="debug"
+```
+
+Make sure to unset this logging level after the issue is resolved, it is quite
+verbose and likely will overload systems if left on indefinitely.
+
+## Missing Spans
+
+There are many points were spans may be dropped. Follow these steps to ensure
+they are not being dropped by the `SDK`.
+
+1. [Enable debug logging](#enable-debug-logging). This should log the `SDK`
+   configuration and all spans sent to registered `SpanProcessor`s. Verify the
+   correct number of span are reported as exported in the log messages.
+   Additionally, pay attention to any `total_dropped` count.
+2. If an incorrect number of spans are being reported as exported, verify you
```

Comment:
```suggestion
2. If an incorrect number of spans are being reported as exported, verify you
```

What determines a "correct" number of spans? Is that figure available somewhere as a grand total? Trying to understand how the user can easily carry out that mental math.

Reply:
It will entirely depend on the users application. I expect that if they are looking at the section for missing spans they will have some idea of what number they expect.

Reply:
Could you add a suggestion here as to how could user check on this?